com.jetbrains.Toolbox.yml

```diff
@@ -0,0 +1,62 @@
+id: com.jetbrains.Toolbox
+command: jetbrains-toolbox
+runtime: org.freedesktop.Platform
+runtime-version: "24.08"
+sdk: org.freedesktop.Sdk
+tags:
+  - proprietary
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --device=dri
+  - --socket=x11
+  - --share=network
+  - --talk-name=org.kde.StatusNotifierWatcher
+  - --filesystem=home
+modules:
+  - name: toolbox
+    buildsystem: simple
+    build-commands:
+      - install -D apply_extra /app/bin/apply_extra
+      - install -Dm755 jetbrains-toolbox.sh /app/bin/jetbrains-toolbox
+      - install -D -m644 -t ${FLATPAK_DEST}/share/icons/hicolor/scalable/apps/ ${FLATPAK_ID}.svg
+      - install -D -m644 -t ${FLATPAK_DEST}/share/applications/ ${FLATPAK_ID}.desktop
+      - install -D -m644 -t ${FLATPAK_DEST}/share/metainfo/ ${FLATPAK_ID}.appdata.xml
+    sources:
+      - type: script
+        dest-filename: apply_extra
+        commands:
+          - mkdir toolbox/
+          - tar -xzf toolbox.tar.gz --directory=toolbox/ --strip-components=1
+          - rm toolbox.tar.gz
+      - type: file
+        path: com.jetbrains.Toolbox.appdata.xml
+      - type: file
+        path: com.jetbrains.Toolbox.desktop
+      - type: file
+        path: com.jetbrains.Toolbox.svg
```

Comment:
Alternatively, you could retrieve the SVG icon from [here](https://resources.jetbrains.com/storage/products/company/brand/logos/Toolbox_icon.svg). Full example here: https://github.com/flathub/com.jetbrains.DataGrip/blob/master/com.jetbrains.DataGrip.yaml#L93